src/common/lambdaConfig.go

```diff
@@ -0,0 +1,92 @@
+package common
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"gopkg.in/yaml.v3"
+)
+
+var LambdaConf *LambdaConfig
+
+type HTTPTrigger struct {
+	Method string `yaml:"method"` // HTTP method (e.g., GET, POST)
+}
+
+type CronTrigger struct {
+	Schedule string `yaml:"schedule"` // Cron schedule (e.g., "*/5 * * * *")
+}
+
+// TODO: add KafkaTrigger struct
+
+// LambdaConfig defines the overall configuration for the lambda function.
+type LambdaConfig struct {
+	HTTPTriggers []HTTPTrigger `yaml:"http,omitempty"` // List of HTTP triggers
```

Comment:
We should probably group these into a "triggers" subcategory.  Because we will also have another subcategory related to perf soon (per-lambda memory limit, max runtime, etc).